configurations/docker_image_name.txt

```diff
@@ -0,0 +1 @@
+ghcr.io/africasvoices/engagement-data-pipeline:main
```

Comment:
If you need unreleased features I think it's better to use the build attempt tag, or perhaps the branch name if we don't expect the branch to change much. If we use "main", then the requested image could change a lot over time which will make it difficult to re-run this commit in future.